### PR TITLE
Pre dev

### PR DESCRIPTION
**Pull request by CapeSathish**

### CODE REVIEW

File: src/main/java/com/capeelectric/model/ApplicationTypes.java [hunk 1]
```diff
@@ -21,6 +21,9 @@
 	
 	@Column(name = "CODE")
 	private String code;
+	
+	@Column(name = "application_Name")
```
Comment (CapeSathish) [inline src/main/java/com/capeelectric/model/ApplicationTypes.java:25]: CHange the name of the column

File: src/main/resources/data.sql [hunk 2]
```diff
@@ -16,14 +16,14 @@ user_type varchar(255),
 updated_date datetime,
 primary key (user_id));
 
-create table APPLICATIONTYPES(id integer not null, application varchar(255), code varchar(20), primary key(id))
+create table APPLICATIONTYPES(id integer not null, application varchar(255), code varchar(20), primary key(id),application_Name varchar(10));
```
Comment (CapeSathish) [inline src/main/resources/data.sql:19]: change the name of the application_Name column

**Review by CapeSathish — COMMENTED** [review]
@thirumoorthy2005 Please change the which i have asked

**Comment by thirumoorthy2005** [comment]
instead of all Boolean  type changed to string type

**Pull request merged**